docs/Clients.md

```diff
@@ -18,7 +18,7 @@ While it is possible and simple to use the raw Redis commands API, in most cases
 | JavaScript | [redis-redisearch][redis-redisearch-url] | [Kyle J. Davis][redis-redisearch-author] | MIT | [![redis-redisearch-stars]][redis-redisearch-url]|
 | TypeScript | [redis-modules-sdk][redis-modules-sdk-url] | [Dani Tseitlin][redis-modules-sdk-author] | BSD-3-Clause | [![redis-modules-sdk-stars]][redis-modules-sdk-url]|
 | C# | [NRediSearch][NRediSearch-url] | [Marc Gravell][NRediSearch-author] | MIT | [![NRediSearch-stars]][NRediSearch-url] |
-| PHP | [redisearch-php][redisearch-php-url] | [Ethan Hann][redisearch-php-author] | MIT | [![redisearch-php-stars]][redisearch-php-url] |
+| PHP | [Redisearch][redisearch-php-url] | [Front][redisearch-php-author] | MIT | [![redisearch-php-stars]][redisearch-php-url] |
```

Comment:
@emmanuelkeller his license is not clear

Reply:
@gkorland The license is MIT

It is in the composer file:
https://github.com/front/redisearch/blob/master/composer.json

Not that visible,  but it appears in the packagist page:
https://packagist.org/packages/front/redisearch